README.md

```diff
@@ -344,7 +348,7 @@ seconds.
 
 #### `EthereumTester.lock_account(account)`
 
-Locks the provide account.  
+Locks the provide account.
```

Comment:
If we keep these whitespace changes maybe go ahead and correct this type I just noticed too?

```suggestion
Locks the provided account.
```